ci_framework/roles/os_must_gather/tasks/main.yml

```diff
@@ -40,6 +40,7 @@
       environment:
         KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
         PATH: "{{ cifmw_path }}"
+        SOS_EDPM: "all"
```

Comment:
This is not enough, because the env variable has to be passed to the actual container, so in our case we need to modify the call like we mention in the[ openstack-must-gather readme](https://github.com/openstack-k8s-operators/openstack-must-gather/blame/b535eb2786d5c052db3e40ce713d576bd534361e/README.md#L36):

```
          oc adm must-gather --image {{ cifmw_os_must_gather_image }}
          --timeout {{ cifmw_os_must_gather_timeout }}
          --host-network={{ cifmw_os_must_gather_host_network }}
          --dest-dir {{ cifmw_os_must_gather_output_dir }}/logs -- SOS_EDPM=$SOS_EDPM gather > {{ cifmw_os_must_gather_output_dir }}/logs/os_must_gather.log
```

As an unrelated note, I think we need to change the redirection to also include stderr